sdk/monitor/azure-monitor-spring-native/src/main/java/com/azure/monitor/applicationinsights/spring/AzureTelemetryActivationConfig.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.applicationinsights.spring;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration(proxyBeanMethods = false)
+public class AzureTelemetryActivationConfig {
+
+    @Bean
+    public AzureTelemetryActivation azureTelemetryActivation(@Value("${applicationinsights.native.spring.non-native.enabled:}") boolean enableEvenWithNonNative) {
```

Comment:
let's remove this property now that we are going with `azure-monitor-spring-native` for the artifact name, I think the artifact name is specific enough to be clear what the intention is